Correctly compare Bazel version numbers.

DrMarcII opened the pull request:
https://github.com/bazelbuild/bazel/issues/4425

jin commented:
This is now implemented at Skylib HEAD:

```python
git_repository(
    name = "bazel_skylib",
    remote = "https://github.com/bazelbuild/bazel-skylib",
    commit = "34d62c4490826f7642843e0617d7fa614994ef79",
)

load("@bazel_skylib//:lib.bzl", "versions")
versions.check("0.5.0")
```

DrMarcII commented:
I have switched to skylib for the version check.

Also updated the version to 0.9.0 as I haven't tested this with any earlier versions at this point.
